### PR TITLE
PY: Fix syntax errors

### PR DESCRIPTION
**Pull request by davidfischer-ch**

### CODE REVIEW

**Comment by davidfischer-ch** [comment]
One question still, how to run tests?

I mean I only issued ./go test_py and it look like some preliminary preprocessing is required, because of:

    Finished processing dependencies for selenium==2.46.0
      File "build/bdist.linux-x86_64/egg/selenium/test/selenium/webdriver/browser_specific_template.py", line 21
        from selenium.##PACKAGE_NAME## import ##GENERAL_FILENAME##
                                                                 ^
    SyntaxError: invalid syntax

      File "/home/david/github/CloudNcodeInc/selenium/build/python/lib/python3.4/site-packages/selenium-2.46.0-py3.4.egg/selenium/test/selenium/webdriver/browser_specific_template.py", line 21
        from selenium.##PACKAGE_NAME## import ##GENERAL_FILENAME##
                                                                 ^
    SyntaxError: invalid syntax

**Comment by AutomatedTester** [comment]
What does the output say after that? It should carry on and run tests...



—

**Comment by ddavison** [comment]
additionally, have you signed the [CLA](http://goo.gl/qC50R)?

**Comment by davidfischer-ch** [comment]
I just signed the CLA.

Here are the latest lines (I install pytest with pip2 and pip3):

    build/python/bin/python setup.py build
    running build
    running build_py
    running egg_info
    writing dependency_links to selenium.egg-info/dependency_links.txt
    writing selenium.egg-info/PKG-INFO
    writing top-level names to selenium.egg-info/top_level.txt
    reading manifest file 'selenium.egg-info/SOURCES.txt'
    reading manifest template 'MANIFEST.in'
    writing manifest file 'selenium.egg-info/SOURCES.txt'
    build/python/bin/py.test build/lib/selenium/test/selenium/webdriver/firefox/ -k="-ignore_firefox" --junitxml=build/test_logs/python-ff-1436531456.xml
    /bin/sh: 1: build/python/bin/py.test: not found
    go aborted!
    Command failed with status (127): [build/python/bin/py.test build/lib/seleniu...]

    (See full trace by running task with --trace)

**Comment by AutomatedTester** [comment]
Looks like I fixed the return statement issue a few days ago but forgot to push. I have landed in the tree. I will land the typo fix shortly.

File: py/selenium/webdriver/remote/remote_connection.py [hunk 1]
```diff
@@ -142,7 +142,7 @@ def get_timeout(cls):
         :Returns:
         Timeout value in seconds for all http requests made to the Remote Connection
         """
-        return None if cls._timeout == socket._GLOBAL_DEFAULT_TIMEOUT or cls._timeout
+        return None if cls._timeout == socket._GLOBAL_DEFAULT_TIMEOUT else cls._timeout
```
Comment (cgoldberg) [inline py/selenium/webdriver/remote/remote_connection.py]: The else doesn't return anything?

**Comment by AutomatedTester** [comment]
landed in 7d6119cac3a0fa2f01513d16a4a32d38e3e197b2

**Pull request closed without merging**